scripts/lib/api/conversionPipeline.ts

```diff
@@ -43,6 +43,33 @@ export async function runConversionPipeline(
   markdownPath: string,
   pkg: Pkg,
 ) {
+  const [files, maybeObjectsInv] = await determineFilePaths(htmlPath, pkg);
+  let initialResults = await convertFilesToMarkdown(
+    pkg,
+    htmlPath,
+    markdownPath,
+    files,
+  );
+
+  await createNecessaryDirs(initialResults);
```

Comment:
I don't think this is actually necessary. But I didn't want to make semantic changes in this PR. I'll try to remove it in a follow up.